### ISSUE TITLE
disable-line affects "global" disable behavior for following lines

### ISSUE THREAD

**Issue by saxx** [palantir/tslint#1624]
### Bug Report
- **TSLint version**: `^3.15.1`
- **TypeScript version**: `2.0.3`
- **Running TSLint via**: (please choose one) `grunt-tslint` and `TSLint for Visual Studio Code`
#### TypeScript code being linted

``` ts
//tslint:disable:no-string-literal 

class MyClass {}
const myClass = new MyClass();

// no linting problems, because of "global" disable
myClass["xxx"]();

// no linting problems because of "global" or "line" disable
myClass["xxx"]();  //tslint:disable-line:no-string-literal 

// now the linter tells me that object access via string literals is disallowed
myClass["xxx"](); 
```
#### Actual behavior

The `disable-line` tslint statement also changes the behavior of the `disable` tslint statement. So, for all following lines it's as if `tslint:enable` has been called.

True, in my code example above there's actually no need for `tslint:disable-line`, because it's already disabled by the "global" `disable`. So this behavior won't happen a lot in most use cases, but it just bit me (I copied/pasted a line with `disable:line`).
#### Expected behavior

I would expect that  `disable-line` really only affects the behavior of the current line. I don't want my "global" `tslint:disable` being re-enabled by `tslint:disable-line`.
